project.clj

```diff
@@ -15,24 +15,33 @@
                  [com.cemerick/url "0.1.1"]
                  [com.datadoghq/java-dogstatsd-client "2.4"]
                  [com.fasterxml.jackson.core/jackson-databind "2.9.9"]
-                 [com.novemberain/langohr "5.2.0" :exclusions [org.clojure/clojure]]
+                 [com.novemberain/langohr "5.2.0" :exclusions [org.clojure/clojure org.slf4j/slf4j-api]]
                  [com.taoensso/nippy "3.1.1"]
                  [io.dropwizard.metrics5/metrics-core "5.0.0" :scope "compile"]
                  [medley "1.3.0" :exclusions [org.clojure/clojure]]
                  [mount "0.1.16"]
-                 [io.jaegertracing/jaeger-core "1.6.0"]
-                 [io.jaegertracing/jaeger-client "1.6.0"]
+                 [io.jaegertracing/jaeger-core "1.6.0" :exclusions [org.slf4j/slf4j-api]]
+                 [io.jaegertracing/jaeger-client "1.6.0" :exclusions [org.jetbrains.kotlin/kotlin-stdlib-common
+                                                                      org.slf4j/slf4j-api]]
                  [org.apache.httpcomponents/fluent-hc "4.5.13"]
-                 [org.apache.kafka/kafka-clients "2.8.2" :exclusions [org.slf4j/slf4j-log4j12 log4j]]
-                 [org.apache.kafka/kafka-streams "2.8.2" :exclusions [org.slf4j/slf4j-log4j12 log4j]]
+                 [org.apache.kafka/kafka-clients "3.7.0" :exclusions [log4j
+                                                                      org.lz4/lz4-java
+                                                                      org.slf4j/slf4j-api
+                                                                      org.slf4j/slf4j-log4j12]]
+                 [org.apache.kafka/kafka-streams "3.7.0" :exclusions [log4j
+                                                                      org.lz4/lz4-java
+                                                                      org.slf4j/slf4j-api
```

Comment:
why bump up client and stream version ? 
Do we know the backward compatibility with this and changelog ?

Reply:
This has been reverted.